applications/StructuralMechanicsApplication/structural_mechanics_application.cpp

```diff
@@ -405,6 +405,7 @@ void KratosStructuralMechanicsApplication::Register() {
     KRATOS_REGISTER_VARIABLE(PRESCRIBED_DISPLACEMENT)
 
     // Constitutive laws variables
+    KRATOS_REGISTER_VARIABLE(LAYER_EULER_ANGLES)
```

Comment:
see my comment in the other PR, please refactor the CL-vars into a separate file

Reply:
Yes! I saw your comment. The thing is that I agree but I wouldn't do that in one of this PR's. If you want I can create an issue and discuss this. Once we all agree I can implement this refactor. @RiccardoRossi

Reply:
there is already #3874
but nothing happened in almost 1.5 years

Reply:
in any case we can do it in a different PR, I will release my block once someone else (@loumalouomega) took a closer look

Reply:
I agree with this PR, but the anisotropic PR must be merged first